assets/js/share.js

```diff
@@ -0,0 +1,24 @@
+
+(function($, undefined) {
+    "use strict";
+    var shareModal = $("#share-to-mastodon");
+    var shareButton = shareModal.find("button");
+
+    shareModal.find("input[name='instance']").on("change paste keyup", function() {
+        // start by chars (https://)
+        if(this.value.startsWith("https://")) {
```

Comment:
`shareButton.attr("disabled", this.value.startsWith("https://"));` d'ailleurs c'est pas plutôt `prop` que `attr`?

Reply:
bien vu.

Reply:
N'oublie pas le '!' devant, vu que c'est inversé


```js
    $shareModal.find("input[name='instance']").on("change paste keyup", function() {
        // start by chars (https://)
        const startsWithHttp = this.value.startsWith("https://")

        $shareButton.prop("disabled", !startsWithHttp);
    });
```